pom.xml

```diff
@@ -227,8 +227,8 @@
         <updatePolicy>daily</updatePolicy>
         <checksumPolicy>fail</checksumPolicy>
       </snapshots>
-      <id>sonatype-nexus-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <id>apache-snapshots</id>
+      <url>https://repository.apache.org/content/groups/snapshots</url>
```

Comment:
We need it if we want fluo-recipes build in TravisCI and depend on a snapshot version of Fluo.  However, that might not be necessary for awhile so we could remove it until then.